safe_transaction_service/history/serializers.py

```diff
@@ -234,7 +235,14 @@ def validate(self, data):
 
     def save(self, **kwargs):
         safe_tx_hash = self.validated_data["contract_transaction_hash"]
-        origin = self.validated_data["origin"]
+        # Origin field on db is a JsonField
+        if self.validated_data["origin"] is not None:
+            try:
+                origin = json.loads(self.validated_data["origin"])
+            except ValueError:
+                origin = json.loads(json.dumps(self.validated_data["origin"]))
```

Comment:
```suggestion
                origin = self.validated_data["origin"]
```

This is the same of what you are writting

Reply:
Done [here](https://github.com/safe-global/safe-transaction-service/pull/1177/commits/36e2571f462ce4fda01edc71d867f1b69e3a7dc0)